app/controllers/label_templates_controller.rb

```diff
@@ -33,17 +33,28 @@ def show
   end
 
   def create
-    label_template = ZebraLabelTemplate.default.save!
-
-    redirect_to label_template_path(label_template, new_label: true)
+    ActiveRecord::Base.transaction do
+      label_template = ZebraLabelTemplate.default
+      label_template.team = current_team
+      label_template.save!
+      log_activity(:label_template_created, label_template)
+      redirect_to label_template_path(label_template, new_label: true)
+    end
+  rescue StandardError => e
```

Comment:
Should we just rescue ActiveRecord::RecordInvalid here?

Reply:
Nope, this is for general error

Reply:
Would it make sense to at least include the stack trace then? As this can be difficult to debug if just looking at server logs.

Reply:
Done